src/integration/utils/runTests/index.js

```diff
@@ -40,24 +42,32 @@ const buildApp = () =>
   });
 
 const startApp = () => {
+  const portNumber = argv.nextJS ? 7081 : 7080;
   return new Promise(resolve => {
     const child = exec(
       `yarn ${
         isDev ? 'dev' : 'start'
-      } & ./node_modules/.bin/wait-on -t 20000 http://localhost:7080/status`,
+      } & ./node_modules/.bin/wait-on -t 20000 http://localhost:${portNumber}/status`,
     );
 
     child.on('exit', resolve);
   });
 };
 
+const runReactTests = () =>
```

Comment:
```suggestion
const runExpressTests = () =>
```
If I'm understanding this correctly this runs the existing test for the express app, this may read better?